src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationService.java

```diff
@@ -101,13 +100,27 @@ private boolean isFullAdmission(ResponseType responseType) {
         map.put("responseDeadline", formatDate(claim.getResponseDeadline()));
         map.put("fourteenDaysFromNow", formatDate(now().plusDays(14)));
 
+        System.out.println("In wrapInMap - 2");
+
         if (response.getResponseType() == FULL_ADMISSION) {
             FullAdmissionResponse fullAdmissionResponse = (FullAdmissionResponse) response;
+            map.put("responseType", "full admission");
             map.put("paymentOption", fullAdmissionResponse.getPaymentIntention().getPaymentOption());
             map.put("paymentOptionDescription", fullAdmissionResponse.getPaymentIntention()
                 .getPaymentOption().getDescription().toLowerCase());
         }
 
+
+        if (response.getResponseType() == PART_ADMISSION) {
+            PartAdmissionResponse partAdmissionResponse = (PartAdmissionResponse) response;
+            map.put("responseType", "partial admission");
+            partAdmissionResponse.getPaymentIntention().ifPresent(paymentIntention ->
+                map.put("paymentOption", paymentIntention.getPaymentOption()));
```

Comment:
do we need `paymentOption` for new email contents? please remove for full admission as well

Reply:
Don't see `paymentOption` being used any more, please remove